### PR TITLE
[docs] fix broken link

### PR DESCRIPTION
**Pull request by eeshugerman**
Issue:

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@eeshugerman thanks for the pull request. I'm going to check this and if it's all good, more than glad to merge it.

Do you mind waiting a bit while i check it?

Stay safe

File: docs/configure/styling-and-css.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If your component files import their own CSS, Storybook’s webpack config will
 - If you are using a CSS precompiler, you may need to add a preset (such as the [SCSS preset](https://github.com/storybookjs/presets/tree/master/packages/preset-scss), or add a loader to Storybook’s webpack config).
 - In Angular, you'll need to take special care how you handle CSS:
 
-  - Either [customize your webpack config](#extending-storybooks-webpack-config)
+  - Either [customize your webpack config](./webpack.md#extending-storybooks-webpack-config)
```
Comment (ndelangen) [inline docs/configure/styling-and-css.md:18]: This link would not work on our docs site.. Thee markdown files are used to generate https://storybook.js.org/

Comment (eeshugerman) [reply docs/configure/styling-and-css.md:18]: oops, yeah i didn't mean to include `.md` there. will update

**Review by shilman — APPROVED** [review]
LGTM

**Pull request merged**